Cheatsheet.md

```diff
@@ -0,0 +1,504 @@
+console.log(a); 
+console.log(b); 
+console.log(c); 
+console.log(d); 
+ ```
+*Output:*
+    
+    geek
+    eeksf
+    forgeeks
+    geeksforgeeks
+
+### [2. split](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/split)
+
+**Description:**
+Used to split a string into an array of substrings, and returns the new array.
+
+**Syntax:**
+ * _string.split(separator, limit)_
+
+| Parameter | Description |
+| :-------- | :---------- |
+|separator	| Optional. Specifies the character, or the regular expression, to use for splitting the string. If omitted, the entire string will be returned (an array with only one item)|
+| limit	| Optional. An integer that specifies the number of splits, items after the split limit will not be included in the array |
+
+**Return value:**
+Array containing splitted values.
+
+**Example:**
+```
+let str = 'It iS a 5r&e@@t Day.'
+let array = str.split(" ");
+console.log(array);
+```
+*Output:*
+
+    [It,iS,a,5r&e@@t,Day.]
+
+### [3. toLowerCase](https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Global_Objects/String/toLowerCase)
+
+**Description:**
+Converts a string to lowercase letters.
+
+**Syntax:**
+ * _string.toLowerCase()_
+
+**Return value:**
+A string, representing the value of a string converted to lowercase.
+
+**Example:**
+```   
+const str = 'It iS a Great Day.';
+const string = str.toLowerCase();
+console.log(string);
+```
+*Output:*
+
+    it is a great day.
+
+### [4. trim](https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Global_Objects/String/trim)
+
+**Description:**
+Removes whitespace from both sides of a string.
+
+**Syntax:**
+ * _string.trim()_
+
+**Return value:**
+String, representing the string with removed whitespace from both ends
+
+**Example:**
+```
+const str = "GeeksforGeeks      ";
+const st = str.trim();
+console.log(st);
+```
+*Output:*
+   
+    GeeksForGeeks
+
+### [5. charAt](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/charAt)
+
+**Description:**
+Returns the character at the specified index in a string.
+
+**Syntax:**
+ * _string.charAt(index)_
+
+**Return value:**
+String, representing the character at the specified index, or an empty string if the index number is not found
+**Example:**
+```
+const str = 'JavaScript is object oriented language';
+console.log(str.charAt(9));
+```  
+  *Output:*
+
+    t
+    
+## Many forms of for loops
+
+### [1. for...in](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/for...in)
+
+**Description:**
+ Loops through the properties of an object. The block of code inside the loop will be executed once for each property.
+
+**Syntax:**
+ * _for (var in object) {<br>
+  code block to be executed<br>
+ }_
+
+| Parameter | Description |
+| :---      | :---        |
+|var	|Required. A variable that iterates over the properties of an object|
+|object	| Required. The specified object that will be iterated|
+

```

Comment:
These are language constructs, not functions. This kind of doc isn't necessary. Please remove this